src/spec/index/handler.spec.js

```diff
@@ -131,11 +131,11 @@ describe('index/handler.js', () => {
 				}),
 				handleConfig = {
 					schema,
-					callback: spy
+					callback: stub
 				};
 
 			config.transport.subscribe.callsFake(obj => {
-				obj.handler(toBuffer(compileFromSchemaDefinition({
+				const result = obj.handler(toBuffer(compileFromSchemaDefinition({
```

Comment:
OPTIONAL: This stub seems to have more logic than it really needs. But that's how it was before. I'd have been tempted to use
```javascript
const content = {
    //the final result of toBuffer(compileFromSchemaDefinition(..),
    //rather than calculating during the test unless it's horrible to read
};
config.transport.subscribe.yieldsTo('handler', content);
```